apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -13,6 +13,40 @@ export function getManifest(appType, locale = 'en_us') {
   );
 }
 
+/* Returns the english default sprite list of SpriteLab
```

Comment:
nit: this can be a single-line comment (`//` instead of `/**/`)

wordsmithing: maybe "Returns the list of default sprites in Spritelab in English"?

Reply:
(similar wordsmithing for the POST request below)